src/test/java/uk/gov/hmcts/reform/finrem/caseorchestration/helper/LetterAddressHelperTest.java

```diff
@@ -0,0 +1,45 @@
+package uk.gov.hmcts.reform.finrem.caseorchestration.helper;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.junit.Test;
+import uk.gov.hmcts.reform.ccd.client.model.CallbackRequest;
+import uk.gov.hmcts.reform.ccd.client.model.CaseDetails;
+
+import java.io.InputStream;
+import java.util.Map;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.junit.Assert.assertThat;
+import static uk.gov.hmcts.reform.finrem.caseorchestration.OrchestrationConstants.LINE_SEPARATOR;
+
+public class LetterAddressHelperTest {
+
+    private ObjectMapper mapper = new ObjectMapper();
+
+    private LetterAddressHelper letterAddressHelper = new LetterAddressHelper();
+
+    private CaseDetails caseDetails() throws Exception {
+        try (InputStream resourceAsStream =
+                     getClass().getResourceAsStream("/fixtures/bulk-print.json")) {
+            return mapper.readValue(resourceAsStream, CallbackRequest.class).getCaseDetails();
+        }
+    }
+
+    @Test
+    public void testAddressIsCorrectlyFormatterForLetterPrinting() throws Exception {
```

Comment:
what about scenarios:
1. address = null
2. one of address fields null
3. address is a map, but all fields are null
4. address is a map, but all fields are empty